ckanext/canada/triggers.py

```diff
@@ -280,19 +282,21 @@ def update_triggers():
                 IF NEW.record_created IS NULL THEN
                     NEW.record_created := OLD.record_created;
                 END IF;
-                IF NEW.record_modified IS NULL THEN
+                IF NOT sysadmin AND NEW.record_modified IS NULL THEN
                     NEW.record_modified := OLD.record_modified;
                 END IF;
-                IF (req_user_modified = '') IS NOT FALSE THEN
+                IF NOT sysadmin AND (req_user_modified = '') IS NOT FALSE THEN
```

Comment:
This shouldn't be necessary. `req_user_modified` is forced to NULL when NOT sysadmin, only sysadmins may set a user_modified value.